packages/pn-commons/src/components/NotificationDetail/NotificationDetailDocuments.tsx

```diff
@@ -32,13 +32,13 @@ const NotificationDetailDocuments = ({
   // TODO: remove comment when link ready downloadFilesLink
 }: Props) => {
 
-  const mapOtherDocuments = (documents: Array<NotificationDetailDocument>) => documents.map((d) => {
+  const mapOtherDocuments = (documents: Array<NotificationDetailDocument>) => documents.map((d, index) => {
```

Comment:
The use of the index is not recommended. Into the object from the be and in the timeline, there is an id or something similar?